example/index.html

```diff
@@ -148,26 +124,20 @@
         stormpath.on('loginError', function (error) {
           console.error(error);
         });
-      }
-
-      window.onload = onDocumentReady;
+      });
     </script>
   </head>
-
   <body>
     <div class="header">
-        <div class="home-menu pure-menu pure-menu-horizontal pure-menu-fixed">
-            <a class="pure-menu-heading" href="">Demo</a>
-        </div>
+      <div class="home-menu pure-menu pure-menu-horizontal pure-menu-fixed">
+        <a class="pure-menu-heading" href="">Demo</a>
+      </div>
     </div>
 
     <div class="content-wrapper">
       <div class="content">
-
         <h1 class="splash-head">Stormpath Widget</h1>
 
-        <div class="loader" id="loading-indicator">Loading...</div>
```

Comment:
We had this in here because there's a short delay after the page is reloaded before the token is verified. It gives the user a little feedback that something is happening.